auth0_flutter_platform_interface/lib/src/credentials.dart

```diff
@@ -54,18 +54,30 @@ class Credentials {
     required this.tokenType,
   });
 
+  static DateTime parseArabicDate(final String arabicDateString) {
```

Comment:
As mentioned in another comment, let's move this out into its own utility, something like `lib/src/date_utils.dart`

Reply:
`parseArabicDate` moved into a separate helper class. To `lib/src/date_utils.dart` location.